website/docs/essays/tx-pool-rbf.md

```diff
@@ -37,7 +45,7 @@ There are several check rule for RBF:
 
```

Comment:
2. The new transaction must not include new, unconfirmed inputs. Specifically, new transaction must not have any `cell_deps` inputs that are outputs of old transactions.